### PR TITLE
Support adding cache=0 in query string param search endpoint config.

### PR DESCRIPTION
**Pull request by olamothe**
https://coveord.atlassian.net/browse/JSUI-2307

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: src/ui/FieldValue/FieldValue.ts [hunk 1]
```diff
@@ -410,6 +410,7 @@ export class FieldValue extends Component {
       const label = selected ? l('RemoveFilterOn', renderedFacetValue) : l('FilterOn', renderedFacetValue);
       new AccessibleButton()
         .withTitle(label)
+        .withLabel(label)
```
Comment (samisayegh) [inline src/ui/FieldValue/FieldValue.ts]: Why is this addition needed?

File: src/utils/UrlUtils.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ export class UrlUtils {
       const mapped = paired.map(pair => {
         const [key, value] = pair;
 
-        if (!value || !key) {
+        if (UrlUtils.valueShouldBeRemovedFromQueryString(value) || UrlUtils.valueShouldBeRemovedFromQueryString(key)) {
```
Comment (samisayegh) [inline src/utils/UrlUtils.ts]: I think it would be better if the name of the utility function remains agnostic to what should be done. Something along the lines of `isValidQueryStringValue()`? In this particular instance, the result of the call will be to remove it because we are inside the `normalizeQueryString` method, but if we were to use the method in another place, maybe the action would be to skip parsing the value, rather than remove it.

File: src/ui/FieldValue/FieldValue.ts (hunk 1 above)
Comment (olamothe) [reply src/ui/FieldValue/FieldValue.ts]: Complains in the console that there is no label.
